rt_rubocop_defaults.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.description   = "rubocop defaults used at runtastic"
   spec.license       = "MIT"
   spec.homepage      = "https://github.com/runtastic/rt_rubocop_defaults"
-  spec.required_ruby_version = "~> 2.5"
```

Comment:
Which kind of change is this? I guess a minor change.
What do you think?

Reply:
would say it's even a patch change, as it's a backwards-compatible fix ...

Reply:
yes patch is fine